BackEnd/src/main/kotlin/quest/marketstack/tradingapp/controller/ClosedTradeController.kt

```diff
@@ -0,0 +1,17 @@
+@file:Suppress("ktlint:standard:no-wildcard-imports")
+
+package quest.marketstack.tradingapp.controller
+
+import org.springframework.web.bind.annotation.*
+import quest.marketstack.tradingapp.model.ClosedTrade
+import quest.marketstack.tradingapp.service.closedtrades.ClosedTradeServiceInterface
+
+@RestController
+@RequestMapping("\${api.closed-trade.base-url}")
```

Comment:
If you see, there's no longer the path of the endpoint hardcoded here. That's because you don't normally want endpoint paths written directly into the code. You want that stuff defined in the application.yml. See the comment I left in the application.yml file for an in-depth explanation